roles/confluent.zookeeper/defaults/clientPort: 2181

```diff
@@ -27,6 +27,7 @@ zookeeper:
   systemd_override: /etc/systemd/system/confluent-zookeeper.service.d
   keytab: /etc/security/keytabs
   properties:
+    clientPort: 2181
```

Comment:
Negative. The client port is not only used to deploy ZooKeeper, but also to configure services
that use ZK. It is actually already defined here:
https://github.com/confluentinc/cp-ansible/blob/5.3.1-post/group_vars/all/shared_vars.yml#L3